spec/offchain.tex

```diff
@@ -171,9 +171,9 @@ \subsubsection{Processing transactions off-chain}
 party $\party_j$, the receiver $\party_i$ checks that $s$ is the next snapshot
 number and that party $\party_j$ is responsible for leading its
 creation.\todo{define $\hpLdr$} Party $\party_i$ then waits until the previous
-snapshot is confirmed ($\bars = \hats$) and all requested transactions
-$\mT_{res}$ are applicable to $\barmU$. % XXX: this is pointless as we would
-% have all the transaction submitted and needs to be applicable
+snapshot is confirmed ($\bars = \hats$), all requested transactions
+$\mT_{res}$ are applicable to $\barmU$, and all requested transactions have
+been seen ($\mT_{res} \subseteq \hatmT$).
```

Comment:
There is also the `spec/fig_offchain_prot.tex` with the composite pseudo code which needs updating.

Reply:
This needs slight updating as we should now mention that "requested transaction ids have been seen and the corresponding transactions are applicable"